### PR TITLE
Add Resources page for ruby

### PR DESCRIPTION
**Pull request by svrnm**
A basic page that explains how resources get detected with ruby (most text is copied from the gem documentation)

Preview: https://deploy-preview-4154--opentelemetry.netlify.app/docs/languages/ruby/resources/

### CODE REVIEW

File: content/en/docs/languages/ruby/resources.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+{{% docs/languages/resources-intro %}}
+
+## Resource Detection
+
+The
+[opentelemetry-resource-detectors](https://rubygems.org/gems/opentelemetry-resource_detectors)
+gem provides a means of retrieving a resource for supported environments
+following the
+[resource semantic conventions](https://opentelemetry.io/docs/specs/semconv/resource/).
```
Comment (svrnm) [inline content/en/docs/languages/ruby/resources.md:14]:
```suggestion
[resource semantic conventions](/docs/specs/semconv/resource/).
```

**Review by kaylareopelle — COMMENTED** [review]
I'm having trouble finding where the resource detectors gem is defined. I see it on RubyGems, but I can't find the source code. 

Could you share the link to the gem documentation you used?

**Comment by svrnm** [comment]
> I'm having trouble finding where the resource detectors gem is defined. I see it on RubyGems, but I can't find the source code.
> 
> Could you share the link to the gem documentation you used?

https://www.rubydoc.info/gems/opentelemetry-resource_detectors here it is, I didn't double check I was hoping this is linked from the gem

**Comment by arielvalentin** [comment]
The all-in-one detectors gems has been deprecated.

Each detector is now in its own gem:

https://github.com/open-telemetry/opentelemetry-ruby-contrib/tree/main/resources

**Comment by svrnm** [comment]
thanks @arielvalentin, I can update the content accordingly.

**Comment by svrnm** [comment]
I need to take another look into this, I will close this PR for now

**Pull request closed without merging**